code/game/objects/structures/crates_lockers/crates/critter.dm

```diff
@@ -16,6 +16,7 @@
 	door_hinge = 5.5
 	door_anim_angle = 90
 	door_anim_squish = 0.35
+	crateexeption = TRUE
```

Comment:
```suggestion
```

Reply:
This is just here to make sure critter crate does not get a list as it is using a snowflake code instead